src/main/java/seedu/address/model/person/Person.java

```diff
@@ -120,4 +120,7 @@ public String toString() {
         return builder.toString();
     }
 
+    public TimeTable getTimeTable() {
+        return  weekly;
```

Comment:
Double space here will probably trigger stylecheck. Also "weekly" as a timetable name doesn't seem to be the best variable name